kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/ColumnMappingSuite.scala

```diff
@@ -199,6 +201,100 @@ class ColumnMappingSuite extends AnyFunSuite with ColumnMappingSuiteBase {
     assertThat(ColumnMapping.findMaxColumnId(schema)).isEqualTo(12)
   }
 
+  test("convertToPhysicalColumnNames: get physical column name with column mapping disabled") {
+    assert(ColumnMapping.convertToPhysicalColumnNames(
+      new StructType()
+        .add("a", StringType.STRING)
+        .add("b", IntegerType.INTEGER),
```

Comment:
nit: you can define one test schema which has combination of top level columns, single nested and more than one nested columns. Use that schema in all tests with or without CM info.

Reply:
also the negative cases map and list